components/api_request_helper/api_request_helper_unittest.cc

```diff
@@ -101,11 +115,7 @@ class ApiRequestHelperUnitTest : public testing::Test {
 
 TEST_F(ApiRequestHelperUnitTest, SanitizedRequest) {
   std::string expected_sanitized_response =
-#if BUILDFLAG(IS_ANDROID)
-      "{\"id\":1,\"jsonrpc\":\"2.0\",\"result\":1.8446744073709552E19}";
-#else
       "{\"id\":1,\"jsonrpc\":\"2.0\",\"result\":1.8446744073709552e+19}";
-#endif
```

Comment:
Now string body is always a result of JSONWriter::Write, so is platform indepenend.